internal/cliwrapper/cliwrapper.go

```diff
@@ -15,12 +15,19 @@ import (
 type cliWrapper struct {
 	podmanFullPath string
 	logger         log.Logger
+	runtimeContext []string
 }
 
-func NewCliWrapper(fullPath string, logger log.Logger) CliWrapper {
+func NewCliWrapper(fullPath string, logger log.Logger, context string) CliWrapper {
+	// Prepand the podman argument
+	runtimeContext := []string{}
+	if context != "" {
+		runtimeContext = append([]string{"--connection"}, context)
```

Comment:
Although `--connection <name>` is a conventional CLI option representation, a single string `--connection=<name>` should work exactly the same (unless `podman` is using an unconventional parser), and handling this additional option as a single string would simplify your logic.

Reply:
Updated